### DISCUSSION TITLE
Opening graphic (landing page animation) for Pixijs

### DISCUSSION THREAD

**Discussion by ronaldolds7** [pixijs/pixijs#9174] [General]
I would like to know who developed the opening graphic (landing page animation) for Pixijs?  If possible, could you provide a tutorial on how you developed it?  Or provide the code so that I can learn how to design/code my own animation.  Thanks!!!!

**Comment by bigtimebuddy** [comment]
We get this question about once a year. Unfortunately, we've made repeated attempts at finding the source code for that header animation, but we only have the minified version, with no source maps. Also, it happens to be on a very old version of Pixi. I'd love it if someone could reverse-engineer it and create a version for v7. You'd be my hero!
